### ISSUE TITLE
SSL Certificate Invalid (ShareX website)

### ISSUE THREAD

**Issue by Soulflare3** [ShareX/ShareX#910]
When trying to confirm Github Gist text uploader I get the screen below

![Imgur](http://i.imgur.com/JVDUiQj.png?1)

**Comment by DanielMcAssey** [comment]
Its possible your connection is being intercepted either by ISP or Firewall, or worse (virus or other malware), causing Chrome to invalidate the SSL certificate one way or another.

The certificate is valid, as you can see below:

![image](https://cloud.githubusercontent.com/assets/3080979/9288886/004c8022-4355-11e5-9d40-1d344bac90e5.png)

**Comment by Soulflare3** [comment]
I started poking around and have figured out what the real problem is.

The machine I am currently using is an old XP Pro Thinkpad that I reinstalled earlier today. After retrying ShareX's site in Vivaldi, I got a different error message:
![screenshot](http://i.imgur.com/Vh59G1x.png)

After a few quick Google searches it seems XP does not support TLS 1.2, thus the protocol is invalid as far as XP is concerned and the reason for the error messages.

I can deal with this, it's just a test/play machine anyway and not my primary system.
